src/command.ts

```diff
@@ -0,0 +1,88 @@
+import * as Line from "../mod.ts";
+
+/**
+ * The main command of the CLI.
+ */
+export class Command implements Line.Interfaces.ICommand {
+  /**
+   * The command that is used on the command line.
+   */
+  public signature!: string;
+
+  /**
+   * An array of subcommands that this command can execute.
+   */
+  public subcommands: typeof Line.Subcommand[] = [];
+
+  public cli: Line.Cli;
+
+  public arguments: { [k: string]: string } = {};
+
+  public options: { [k: string]: string } = {};
```

Comment:
Don't suppose we could use maps? just for that tiny bit increase of perf?

Reply:
yea it's my plan to optimize this entire project to use maps in all places. key value stores are hella old school now 😏